packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -10,54 +10,30 @@ export class MakeCogGithub extends Github {
     this.imagery = imagery;
   }
 
-  /**
-   * Get the master aerial tileset config from the basemaps config repo
-   */
-  async getTileSetConfig(): Promise<ConfigTileSetRaster> {
-    this.logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    const path = 'config/tileset/aerial.json';
-    const response = await this.octokit.rest.repos.getContent({ owner, repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return JSON.parse(Buffer.from(response.data.content, 'base64').toString());
-    } else {
-      throw new Error('Unable to find the content.');
-    }
-  }
-
   /**
    * Prepare and create pull request for the aerial tileset config
    */
-  async createTileSetPullRequest(
-    layer: ConfigLayer,
-    jira: string | undefined,
-    category: Category,
-  ): Promise<number | undefined> {
+  async updateRasterTileSet(config: string, layer: ConfigLayer, category: Category): Promise<void> {
+    const branch = `feat/config-raster-${this.imagery}`;
```

Comment:
I thought we needed a common prefix to automatically create pull-requests? or is the common prefix `feat/config-`?

Reply:
I decide to use `feat/config-*`

Reply:
could we maybe use `feat/bot-config` so we separate out bot commits vs human commits